local-setup/kustomize/components/platform-mesh-operator-resource/platform-mesh.yaml

```diff
@@ -15,6 +15,12 @@ spec:
   featureToggles:
    - name: "feature-enable-getting-started"
   values:
+    account-operator:
+      enabled: true
+    extension-manager-operator:
+      enabled: true
+    rebac-authz-webhook:
+      enabled: true
```

Comment:
our goal is to reduce the number of overwrites in this resource to the bare minimum required for the local setup. 

I don't think we should merge any value overwrites that repeats a value that is already set in the components chart. We must find another way